tests/end2endtest/es_schema.yml

```diff
@@ -0,0 +1,109 @@
+# Easy-server schema file for pywbem end2end tests
+#
+# This file is used to validate the user-defined sections in the easy-server
+# server file and vault file.
+#
+# Keep this schema in sync with the code in the wbem_connection() fixture that
```

Comment:
Change to json-schema. The word schema is so general that it confuses the  reader.

Reply:
DONE